src/main/java/seedu/address/logic/commands/RemarkCommand.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PATIENTS;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.patient.Patient;
+import seedu.address.model.patient.Remark;
+
+/**
+ * Changes the remark of an existing patient in the address book.
+ */
+public class RemarkCommand extends Command {
+
+    public static final String MESSAGE_ADD_REMARK_SUCCESS = "Added remark to Patient: %1$s";
+    public static final String MESSAGE_DELETE_REMARK_SUCCESS = "Removed remark from Patient: %1$s";
+
+    public static final String COMMAND_WORD = "remark";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the remark of the patient identified "
+            + "by the index number used in the last patient listing. "
+            + "Existing remark will be overwritten by the input.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "r/ [REMARK]\n"
```

Comment:
`[REMARK]` should be without square bracket?

Reply:
Hmm I added the square brackets following the tutorial given in our textbook. I believe they added it in as entering an empty remark serves to remove an existing remark assigned to the same patient. Currently, nurses can use the remark command to both create new and modify existing remarks. 

Will it be better to delegate this task to the edit command instead?

Reply:
Oh that makes sense 👍